spec/models/loan_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe Loan, type: :model do
+  describe "validates Loan" do
+    let(:loan) { Loan.new(loan_amount: "10000", terms: 12, interest_rate: '10', 
+                interest_only: 'Yes', day: DateTime.current.beginning_of_day) }
+
+    context 'When povided all the required filed' do
+      it "creats an instance without any error" do
+        loan = Loan.new(loan_amount: "10000", terms: 12, interest_rate: '10', interest_only: 'Yes', day: DateTime.current.beginning_of_day)
+        expect(loan).to be_valid
+      end
+    end
+
+    context 'When column does not have valid input' do
```

Comment:
You are mixing simple quotes with double quotes? No no no.
The rails guide says that you must always use the simple quotes, unless you are making interpolation:
```
'any normal text'
"any #{variable} text"
```